app/jobs/zipmaker_job.rb

```diff
@@ -9,6 +10,26 @@ class ZipmakerJob < ApplicationJob
   # @param [String] druid
   # @param [Integer] version
   def perform(druid, version)
+    binary_path = Moab::StorageServices.object_version_path(druid, version)
+    zip_path = DruidVersionZip.new(druid, version).file_path
+    unless File.exist?(zip_path)
+      ZipmakerJob.zip_binary(zip_path, binary_path) if binary_path
+    end
     PlexerJob.perform_later(druid, version)
   end
+
+  # @param [String] path to druid version zip
+  # @param [String] path to druid version
+  # @todo calculate md5 of zip for plexer
+  def self.zip_binary(zip_path, binary_path)
```

Comment:
would a better name for this method be ...   create_zip ?     Why "binary"?

Reply:
Or rather, `create_zip!`, since we `raise` unless successful.